doc/api/async_hooks.md

```diff
@@ -76,6 +76,12 @@ function destroy(asyncId) { }
 // (either directly or through other means of resolving a promise).
 function promiseResolve(asyncId) { }
 ```
+### Class: `AsyncHook`
```

Comment:
There is an empty line missing before this.

That being said, this should be moved anyway, to the documentation of `.enable()` and `.disable()`, as those are methods of this class.

Reply:
@addaleax, thanks. Put it in suggested place, please take a look now.